Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1187,7 +1187,7 @@ namespace Babylon
         const auto texture = info[0].As<Napi::External<TextureData>>().Data();
         const auto value = info[1].As<Napi::Number>().Uint32Value();
 
-        texture->AnisotropicLevel = value;
+        texture->AnisotropicLevel = static_cast<uint8_t>(value);
```

Comment:
For cases where the static cast is “narrowing” to a smaller or signed data type, we should consider using gel::narrow instead of static_cast. If we ever have a narrowing bug, this would make it much easier to diagnose.

Reply:
WIP